legacy/definition.go

```diff
@@ -677,7 +677,7 @@ func migrateRuleSet(lang utils.Language, r RuleSet, localization flows.Localizat
 		}
 
 		// resthook rulesets operate on the webhook status, saved as category
-		router = routers.NewSwitchRouter(defaultExit, fmt.Sprintf("@results.%s.category", utils.Snakify(resultName)), cases, "")
+		router = routers.NewSwitchRouter(defaultExit, fmt.Sprintf("@(default(results.%s.category, \"Success\"))", utils.Snakify(resultName)), cases, "")
```

Comment:
this is needed when we follow a resthook action in case it didn't save any results because it didn't have any subscribers